en/README.md

```diff
@@ -0,0 +1,1142 @@
+    |-----------fetchItems(Request)--------------->|
+    |<----------fetchItems(Response)---------------|
+    |<------------putItems(Request)----------------|
+    |-------------putItems(Response)-------------->|
+    |                                              |
+    |-----------fetchHistory(Request)------------->|
+    |<----------fetchHistory(Response)-------------|
+    |<------------putHistory(Request)--------------|
+    |-------------putHistory(Response)------------>|
+    |                                              |
+    |-----------fetchTriggers(Request)------------>|
+    |<----------fetchTriggers(Response)------------|
+    |<-----------putTriggers(Request)--------------|
+    |                Use "ALL"option               |
+    |-----------putTriggers(Response)------------->|
+    |                                              |
+    |-------------fetchEvents(Request)------------>|
+    |<------------fetchEvents(Response)------------|
+    |<------------putEvents(Request)---------------|
+    |-------------putEvents(Response)------------->|
+    |                                              |
+    |--updateMonitoringServerInfo(notification)--->|
+    |                                              |
+
+```
+## Data type
+
+ - This section describes about data types that are defined for Hatohol. They(the data types) use date type that is defined in the JSON-RPC internally.
+
+|Name|JSON type|description|
+|:---|:---------|:---|
+|TimeStamp|string|This type is used to times of day. Time value MUST be stored in UTC. Format is YYYYMMDDhhmmss.nnnnnnnnn. YYYY, MM, DD, hh, mm, ss, and nnnnnnnnn each expresses the A.D., Month, Day, hour, minute, second, nano second. It can abbreviate nano second. If the above or nano second digits is less than nine, it will be inserted zero to fill digit(s)(Ex.100 -> 100.000000000, 100.1234 -> 100.123400000).|
+|Boolean|true, false|This type is used to store boolean value. It can assign true or false to show authenticity of value.|
+|Number|number|This type is used to store number value. It needs to be more than 0 and less than 2147483647.|
+|String255|string|This type is used to store string value. It needs to be the number of characters less than 255 characters.|
+|URI2047|string|This type is used to store string value. It MUST be the number of characters less than 2047. It should satisfy URI requirements which is defined in the RFC3986, RFC6874. And its octet size SHOULD be less than 2047.|
+|String32767|string|This type is used to store string value. It needs to be the number of characters less than 32767 characters.|
+
+In any case, the number of character MUST be counted by UTF-32 code point number after applying NFC nomalization.
+
+## About start up operation
+
+ - Immediatery after starting or restarting Hatohol server and HAP, they exchange usable procedures information with exchangeProfile procedure. There can optimize procedures based on exchanged information.
```

Comment:
-1  呼び出し制限を最適化とは？

↓ではどうでしょうか。

英: These information makes to avoid calling procedure which is not supported each other.
日: これらの情報により、互いにサポートしていないプロシージャの呼び出しを回避する事ができます。

仕様書: この情報を基に，互いが使用するプロシージャの最適化を行うことができます。